kolibri/utils/server.py

```diff
@@ -657,8 +657,8 @@ def start(port=0, zip_port=0, serve_http=True, background=False):
     service_plugin.subscribe()
 
     # Setup zeroconf plugin
-    zeroconf_plugin = ZeroConfPlugin(bus, port)
```

Comment:
Shouldn't commit this change, although I know it's needed for WSL!

Reply:
This was pushed by mistake. I will fix tihs.